dallinger/recruiters.py

```diff
@@ -615,6 +607,30 @@ def open_recruitment(self, n=1):
             "message": "HIT now published to Amazon Mechanical Turk",
         }
 
+    def assign_experiment_qualifications(self, worker_id, qualifications):
+        """Assigns MTurk Qualifications to a worker.
+
+        @param worker_id       string  the MTurk worker ID
+        @param qualifications  list of dict w/   `name`, `description` and
+                               (optional) `score` keys
+        """
+        by_name = {qual["name"]: qual for qual in qualifications}
+        result = self._ensure_mturk_qualifications(qualifications)
+        for qual in result["new_qualifications"]:
```

Comment:
Shouldn't this check `available` for these qualifications, since some may not have been available after 5 tries?

Reply:
I meant to come back to this issue and forgot. I can't decide if it should check here, or do something (log an error, I guess?) at the end of `_ensure_mturk_qualifications()`.  Went with the latter.